src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -87,44 +82,42 @@ You can use CSS to customize the look and feel of the dots.
 
 ![](images/Carousel_before_after_2.png)
 
-### Using the Carousel Inside Columns
+### Using the Carousel Pattern inside the Columns Pattern
```

Comment:
This should be in the "Additional Notes" section at the end. Also, it's not clear to which CSS are we adding this - the screen CSS, the widget CSS, the applications global CSS.